bullet_train-super_scaffolding/lib/scaffolding/script.rb

```diff
@@ -183,7 +196,19 @@ def check_required_options_for_attributes(scaffolding_type, attributes, child, p
       puts "Adding new fields to #{child} with '#{generation_command}'".green
     end
     puts ""
-    `#{generation_command}` unless @options["skip-migration-generation"]
+
+    unless @options["skip-migration-generation"]
+      untracked_files = has_untracked_files?(git_status) ? get_untracked_files(git_status) : []
+      generation_thread = Thread.new { `#{generation_command}` }
+      generation_thread.join # Wait for the process to finish.
+
+      newly_untracked_files = has_untracked_files?(git_status) ? get_untracked_files(git_status) : []
+      if (newly_untracked_files - untracked_files).size.zero?
+        # Rails will output an error if nothing was generated,
+        # so we don't have to write anything extra upon exiting.
```

Comment:
Is the Rails error message going to be useful in a Bullet Train context? I think the error that people would see is going to say:

> The name 'Foo' is either already used in your application or reserved by Ruby on Rails. Please choose an alternative or use --skip-collision-check or --force to skip this check and run this generator again.

But I don't think that adding `--skip-collision-check` or `--force` to the `bin/super-scaffold` command is going to work.

Maybe we need to add a message telling people that they could use `--skip-migration-generation`?

Reply:
That's a good point, I'm not sure if I'll be able to prevent the default rails message from being output, but I'll try to at least write a meaningful message.

Reply:
Just pushed https://github.com/bullet-train-co/bullet_train-core/pull/604/commits/39150dd17a6f7a48d75e300ff0237518d44b0a97